devicetypes/smartthings/springs-window-fashions-remote.src/springs-window-fashions-remote.groovy

```diff
@@ -0,0 +1,115 @@
+/**
+ *  Copyright 2015 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "Springs Window Fashions Remote", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.remotecontroller", hidden: true) {
+
+        capability "Battery"
+
+        fingerprint mfr:"026E", prod:"5643", model:"5A31", deviceJoinName: "2 Button Window Remote"
+        fingerprint mfr:"026E", prod:"4252", model:"5A31", deviceJoinName: "3 Button Window Remote"
+    }
+
+    simulator {
+
+    }
+
+    tiles {
+        standardTile("state", "device.state", width: 2, height: 2) {
+            state 'connected', icon: "st.unknown.zwave.remote-controller", backgroundColor:"#ffffff"
+        }
+
+        valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
+            state "battery", label:'batt.', unit:"",
```

Comment:
@greens I keep forgetting to leave this comment, and am surprised that QA hasn't said anything about it, but do we want this to display "batt." or the actual percentage?

Reply:
Ok, ignore this, since I just talked to you and it's all sub optimal. :/